### ISSUE TITLE
`NoClassDefFoundError` when using `detekt-rules-ktlint-wrapper` via CLI (`-p` option) in `2.0.0-alpha.2`

### ISSUE THREAD

**Issue by lish82** [detekt/detekt#9177]
## Expected Behavior

When providing the `detekt-rules-ktlint-wrapper` jar via the `-p` (or `--plugins`) option in the detekt CLI, detekt should successfully load the ktlint rules and execute without an error.
In the stable 1.x versions (e.g., `1.23.8` with `detekt-formatting`), the plugin jar works out of the box because it includes the ktlint classes (Shadow/Fat JAR).

## Observed Behavior

Executing the detekt CLI with `-p detekt-rules-ktlint-wrapper-2.0.0-alpha.2.jar` results in the following runtime error:
```text
java.lang.NoClassDefFoundError: com/pinterest/ktlint/ruleset/standard/rules/AnnotationRule
```
Upon inspecting the contents of `detekt-rules-ktlint-wrapper-2.0.0-alpha.2.jar`, it appears that the ktlint classes are not included. It seems the JAR is no longer built as a shadow JAR, unlike the previous `detekt-formatting` artifact.

## Steps to Reproduce

1. Download the detekt CLI jar for `2.0.0-alpha.2`.
1. Download `detekt-rules-ktlint-wrapper-2.0.0-alpha.2.jar`.
1. Run the CLI with the wrapper plugin, following the official docs (https://detekt.dev/docs/rules/ktlint):
   ```bash
   java -jar detekt-cli-2.0.0-alpha.2-all.jar -p detekt-rules-ktlint-wrapper-2.0.0-alpha.2.jar
   ```
1. Observe the `java.lang.NoClassDefFoundError` in the output.

## Context

I am trying to run detekt with the ktlint rules enabled via CLI. Since the wrapper JAR does not bundle the ktlint dependencies, the CLI execution fails unless I manually download and provide all the ktlint jars in the classpath. 
It is unclear to me whether the shadow JAR packaging was intentionally removed in 2.0.0 or if this is a missing configuration during the migration. This leaves me unsure of the proper way to use the wrapper via CLI.

## Your Environment

* Version of detekt used: `2.0.0-alpha.2`
* Version of Gradle used (if applicable): N/A (Using CLI)
* Gradle scan link (add `--scan` option when running the gradle task): N/A (Using CLI)
* Operating System and version: `macOS Tahoe 26.3.1 (a)`
* Link to your project (if it's a public repository): N/A

**Comment by BraisGabin** [comment]
I see the problem. `detekt-rules-ktlint-wrapper-2.0.0-alpha.2.jar` depends on `ktlint-repackage-2.0.0-alpha.2-all.jar`. Gradle handles that automatically but `cli` doesn't. We should shadow that dependency and publish a detekt-rules-ktlint-wrapper-2.0.0-alpha.2-all.jar

As a workaround, you can add [`ktlint-repackage-2.0.0-alpha.2-all.jar`](https://central.sonatype.com/artifact/dev.detekt/ktlint-repackage) as another plugin and it should work.

**Comment by lish82** [comment]
Thank you for your quick response and for providing me the workaround! It worked perfectly.
